Open_Science_New

```diff
@@ -576,18 +560,18 @@
 
 	  <!-- Cuban Brain Mapping -->
 	  <section data-transition="convex">
-	    <a href="" target="_blank"><img src="images/Cuban_brain_mapping.png" width="120%"></a>
+	    <a href="https://journals.sagepub.com/doi/abs/10.1177/155005941104200303?rfr_dat=cr_pub%3Dpubmed&url_ver=Z39.88-2003&rfr_id=ori%3Arid%3Acrossref.org&journalCode=eegb"><img src="images/Cuban_brain_mapping.png" width="120%"></a>
 	  </section>
 	    <!-- ABIDE -->
 	    <section data-transition="fade">
 	      <table class="reveal">
 		<h2>ABIDE</h2>
 	      <tr>
 		<td width="45%" style="vertical-align: middle">
-		  <img src="images/ABIDE_paper.png">
+		  <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4162310/"><img src="images/ABIDE_paper.png"></a>
```

Comment:
the actual paper is here: 
https://dx.doi.org/10.1038%2Fmp.2013.78

Reply:
thanks! this was removed so N/A